BGL/include/CGAL/boost/graph/boost_parameters_interface.h

```diff
@@ -19,6 +19,6 @@
 
 
 
-CGAL_add_pmp_parameter(vertex_index_t, vertex_index_map, vertex_index_map)
-CGAL_add_pmp_parameter(edge_index_t, edge_index_map, edge_index_map)
-CGAL_add_pmp_parameter(graph_visitor_t, visitor, visitor)
+CGAL_add_named_parameter(vertex_index_t, vertex_index_map, vertex_index_map)
```

Comment:
While you are at renaming the macro, could you follow CGAL guidelines and use a full-uppercase name?